src/features/requestedPackages/components/RequestedPackageList.tsx

```diff
@@ -47,7 +47,7 @@ export const RequestedPackageList = ({
       </StyledAccordionSummary>
       <StyledAccordionDetails sx={{ padding: 0 }}>
         <TableContainer>
-          <Table sx={{ width: "420px", tableLayout: "fixed" }}>
+          <Table sx={{ width: "100%", tableLayout: "fixed" }}>
```

Comment:
I think I previously thought that with `table-layout: fixed`, I needed to set a concrete width in pixels, but the [MDN article on table-layout](https://developer.mozilla.org/en-US/docs/Web/CSS/table-layout) suggests that a percentage value for the width is fine. The key point is two-fold:

1. Must set width to something (cannot leave unspecified)
2. Except must not set width to `auto`

In order for the browser to respect `table-layout: fixed` and not use `table-layout: auto` instead.